rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -28,11 +29,14 @@ local function balance()
   local backend = backends:get(backend_name)
   -- lb_alg field does not exist for ingress.Backend struct for now, so lb_alg
   -- will always be round_robin
-  local lb_alg = backend.lb_alg or "round_robin"
+  local lb_alg = backend.lb_alg or "ewma"
```

Comment:
then extract this to `get_current_lb_alg` and use it in `after_balance` function to decide if you wanna call `ewma.after_balance`